### PR TITLE
[doc][ybm] Partition by region asym node settings

### PR DESCRIPTION
**Pull request by ddhodge**
YBM geopartioning now allows asymmetric node settings
DOC-236

@netlify /preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-geopartition/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-geopartition.md [hunk 1]
```diff
@@ -91,14 +92,15 @@ Fault tolerance is applied to all regions in the cluster, including those added
 
 **Regions** - For each region, choose the following:
 
-- the [region](../../create-clusters-overview/#cloud-provider-regions) where the nodes will be located.
-- the VPC in which to deploy the nodes. Only VPCs using the selected cloud provider and available in the selected region are listed. For multi-region GCP clusters, the same VPC is used for all regions. VPCs must be created before deploying the cluster. Refer to [VPC networking](../../cloud-vpcs/).
-- the number of nodes to deploy in the regions; each region has the same number of nodes.
+- [region](../../create-clusters-overview/#cloud-provider-regions) where the nodes will be located.
+- VPC in which to deploy the nodes. Only VPCs using the selected cloud provider and available in the selected region are listed. For multi-region GCP clusters, the same VPC is used for all regions. VPCs must be created before deploying the cluster. Refer to [VPC networking](../../cloud-vpcs/).
+- number of nodes to deploy in the regions; each region has the same number of nodes.
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-geopartition.md:97]: >each region has the same number of nodes.
This is not true for asymmetric geo clusters. You can specify different # of nodes per region there.

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md [hunk 2]
```diff
@@ -76,10 +76,10 @@ Create a cluster.
...
 | --fault-tolerance | Fault tolerance for the cluster. `NONE`, `NODE`, `ZONE`, or `REGION`. |
-| --node-config | Number of vCPUs, disk size, and IOPS per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
+| --node-config <br> [Deprecated in v2.19] | Number of vCPUs, disk size, and IOPS per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md:79]: `v2.19`? Does that mean the DB version? If so, the DB version isn't related to when we deprecated it. Maybe the version here could be the CLI release version instead?

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md [hunk 3]
```diff
@@ -167,5 +167,5 @@ Update the specified cluster.
...
 | --new-name | The new name for the cluster. |
-| --node-config | Number of vCPUs and disk size per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
-| --region-info | Region details for multi-region cluster, provided as key-value pairs.<br>Arguments:<ul><li>region - name of the region</li><li>num-nodes - number of nodes for the region</li><li>vpc - name of the VPC</li></ul>Specify one `--region-info` flag for each region in the cluster.<br>If specified, region and num-nodes is required, vpc is optional. |
+| --node-config <br> [Deprecated in v2.19] | Number of vCPUs and disk size per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md:170]: Same question about version as above.

**Review by aishwarya24 — APPROVED** [review]
LGTM with a comment

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md [hunk 4]
```diff
@@ -75,11 +75,11 @@ Create a cluster.
...
-| --fault-tolerance | Fault tolerance for the cluster. `NONE`, `NODE`, `ZONE`, or `REGION`. |
-| --node-config | Number of vCPUs, disk size, and IOPS per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
-| --num-faults-to-tolerate | The number of faults to tolerate. 0 for NONE; 1 for ZONE; 1, 2, or 3 for NODE and REGION. Default is 1 (or 0 for NONE). |
+| --fault-tolerance | Fault domain for the cluster. `NONE`, `NODE`, `ZONE`, or `REGION`. |
+| --node-config <br> [Deprecated in v0.1.19] | Number of vCPUs, disk size, and IOPS per node for the cluster, provided as key-value pairs.<br>Arguments:<ul><li>num-cores - number of vCPUs per node</li><li>disk-size-gb - disk size in GB per node</li><li>disk-iops - disk IOPS per node (AWS only)</li></ul>If specified, num-cores is required and disk-size-gb and disk-iops are optional. |
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-cluster.md:79]: This is CLI version

**Review by RP1807 — APPROVED** [review]

**Review by daniel-yb — APPROVED** [review]

**Pull request merged**